src/xcode/ENA.xcworkspace/xcshareddata/swiftpm/Package.resolved

```diff
@@ -154,6 +154,15 @@
           "version": "4.6.0"
         }
       },
+      {
+        "package": "swift-collections",
```

Comment:
This requires an update of the licenses.json in the root of the project followed by `bundle exec fastlane update_licenses`